test/StripeResource.spec.js

```diff
@@ -1023,8 +1023,6 @@ describe('StripeResource', () => {
 
   describe('Request Timeout', () => {
     it('should allow the setting of a request timeout on a per-request basis', (done) => {
-      stripe.setTimeout(1000);
```

Comment:
Why didn't this test start failing?

Reply:
This tests that the request is sent with a timeout of 10 when configured as a per-request option